src/rcms/fm/app/level1/HCALEventHandler.java

```diff
@@ -1038,9 +1042,6 @@ protected void publishRunInfoSummary() {
 
     // TODO JHak put in run start time and stop times. This was always broken.
 
-    Hashtable<String, String> globalRenamedParams = new Hashtable<String, String>();
-    globalRenamedParams.put(  "LOCAL_RUN_KEY"  ,                 "RUN_CONFIG_SELECTED"        );
```

Comment:
DQM still uses this parameter for the task. we should either ask David to implement to move to use `DQM_TASK` or `LOCAL_RUNKEY_NAME` or keep these for now

Reply:
Yes, @DryRun  should certainly move to `DQM_TASK`